protocol/testutil/constants/messages.go

```diff
@@ -68,6 +68,16 @@ var (
 			Amount:    500_000_000, // $500
 		},
 	}
+	Msg_Transfer_Invalid_SameSenderAndRecipient = &sendingtypes.MsgCreateTransfer{
```

Comment:
ValidateBasic is now expected to not be implemented and users should be simulating their transactions before submitting them which is why I swapped to a message where we control the message type explicitly.